app/src/main/java/com/wireguard/android/fragment/TunnelListFragment.kt

```diff
@@ -206,10 +221,28 @@ class TunnelListFragment : BaseFragment() {
             tunnelList.addOnScrollListener(FloatingActionButtonRecyclerViewScrollListener(createFab))
             executePendingBindings()
         }
+        // Collapse searchview on fragment transaction
+        parentFragmentManager.addOnBackStackChangedListener {
+            if (searchItem.isActionViewExpanded) {
+                searchItem.collapseActionView()
+            }
+        }
         return binding?.root
     }
 
+    // Collapse searchview on activity change
+    override fun onPause() {
```

Comment:
On second thought just ditch this entire override since it'll break app -> recents -> app continuity as well...

Reply:
Sure